Yvand.EntraCP/Yvand.EntraClaimsProvider/Configuration/EntraIDProviderConfiguration.cs

```diff
@@ -69,6 +69,13 @@ public interface IEntraIDProviderSettings
         /// Gets if only security-enabled groups should be returned
         /// </summary>
         bool FilterSecurityEnabledGroupsOnly { get; }
+
+        /// <summary>
+        /// Comma separated list of group IDs (max 18 values). If set, users must be member of any of these groups to be returned to SharePoint
```

Comment:
Are there any checks for that (max 18 groups)? I see that setter only assigns the value.

Reply:
added in commit https://github.com/Yvand/EntraCP/pull/243/commits/368a5fed79cfd9bdedec7c11f50c3ddf435c66e3